### PR TITLE
Create 2023 election documents

### LINKED ISSUE #1483: Create the directory on this repository for the election
Use previous years as a template

### PR DESCRIPTION
**Pull request by dyladan**
Fixes #1483

### CODE REVIEW

File: elections/2023/governance-committee-election.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+# Results
+
+Voting will close on 25 October 2023 at 00:00 UTC. Nominees will be stack ranked. If a nominee becomes ineligible (for instance, if more than three topmost nominees work for the same company), the election committee will skip those nominees and pick the nominee with the next-highest score. The exact scores for each candidate will be public.
+
+# Schedule
+
+| Date                       | Activity                                                                                                                                   |
+| -------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------ |
+| June 2023                  | This document announced                                                                                                                    |
+| {{TODO DATE}}              | Election blog post and call for nominations                                                                                                |
```
Comment (dyladan) [inline elections/2023/governance-committee-election.md:99]: @open-telemetry/otel-elections we need to decide on these dates

**Comment by dyladan** [comment]
Link check fails because it points at the candidate list which is created by this PR

File: elections/2023/governance-committee-election.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+# Vacancies
+
+Current governance committee members (alphabetical order):
+
+* [Alolita Sharma](https://github.com/alolita), Apple, until October 2024
+* [Ben Sigelman](https://github.com/bhs), Lightstep, until October 2023
+* [Bogdan Drutu](https://github.com/BogdanDrutu), Splunk, until October 2023
+* [Daniel Dyla](https://github.com/dyladan), Dynatrace, until October 2024
+* [Juraci Paixão Kröhling](https://github.com/jpkrohling), Grafana Labs, until October 2023
+* [Morgan James McLean](https://github.com/mtwo), Splunk, until October 2024
```
Comment (mtwo) [inline elections/2023/governance-committee-election.md:34]:
```suggestion
* [Morgan McLean](https://github.com/mtwo), Splunk, until October 2024
```

**Review by mtwo — APPROVED** [review]

File: elections/2023/governance-committee-election.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+* GitHub handle
+* List either:
+  * Your contributions to OpenTelemetry that make you eligible to vote, or
+  * Your non-code contributions per the [Members of Standing section of the charter](../../governance-charter.md#members-of-standing): particularly, your relationship to a well-known project that's taken a hard dependency on OpenTelemetry, or your involvement with a well-known organization's adoption of OpenTelemetry as an end-user.
+
+All exemption forms are private. Only the current governance and election committee members will have access to this information. The governance committee will discard the lists one month after the election.
+
+# Vote
+
+Everyone with voting rights may log into [Helios Voting](https://vote.heliosvoting.org/helios/elections/1ee70ee4-11ce-11ee-aaf8-0a8c9aac83f9/view) using their GitHub account. Voting will be approval voting, where each voter may select up to five candidates. The five candidates with the most votes win the election.
```
Comment (yurishkuro) [inline elections/2023/governance-committee-election.md:84]: Can anyone login & vote with GitHub or only a predetermined list (of eligible members)?

Comment (jpkrohling) [reply elections/2023/governance-committee-election.md:84]: The election committee will upload a list of github usernames that are eligible for voting.

File: elections/2023/governance-committee-election.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+| {{TODO DATE}} 23:59 UTC    | End of call for nominations                                                                                                                |
+| {{TODO DATE}}              | Nominees ratified, and Governance Committee ensures nominees' commitment                                                                   |
+| {{TODO DATE}}              | Preliminary list of nominees announced                                                                                                     |
+| {{TODO DATE}}              | List of nominees is finalized on [community](https://github.com/open-telemetry/community) GitHub and advertised via mailing list and Slack |
+| {{TODO DATE}} 23:59 UTC    | Last day to apply for a member of standing exemption                                                                                       |
+| 23 Octoboer 2023 00:00 UTC | Voting period begins                                                                                                                       |
```
Comment (jpkrohling) [inline elections/2023/governance-committee-election.md:105]:
```suggestion
| 23 October 2023 00:00 UTC | Voting period begins                                                                                                                       |
```

File: elections/2023/governance-committee-election.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+| {{TODO DATE}}              | Election blog post and call for nominations                                                                                                |
+| {{TODO DATE}} 23:59 UTC    | End of call for nominations                                                                                                                |
+| {{TODO DATE}}              | Nominees ratified, and Governance Committee ensures nominees' commitment                                                                   |
+| {{TODO DATE}}              | Preliminary list of nominees announced                                                                                                     |
+| {{TODO DATE}}              | List of nominees is finalized on [community](https://github.com/open-telemetry/community) GitHub and advertised via mailing list and Slack |
+| {{TODO DATE}} 23:59 UTC    | Last day to apply for a member of standing exemption                                                                                       |
```
Comment (jpkrohling) [inline elections/2023/governance-committee-election.md:104]: I recommend this being a couple of days before the voting period. While it was uneventful to have it the day before in previous elections, I think it's safer to allow for a buffer...

**Comment by dyladan** [comment]
Updated dates. Please take a look to make sure I haven't made any mistakes.

File: elections/2023/governance-committee-election.md [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+* 13 October 2023 - final deadline to submit nominations
+* 16 October 2023 - official nominees list published
+* 23 October 2023 to 25 October 2023 - voting takes place
+* 27 October 2023 - results announced
+
+We highly encourage participation in this election cycle to ensure that the community is well-represented by the Governance Committee.
+
+# TL;DR
+
+* If you've been nominated or are willing to nominate yourself: check the [charter document](../../governance-charter.md) and confirm you are ready for the commitment. Make sure to provide all necessary information before 13 October 2023 00:00 UTC
```
Comment (jpkrohling) [inline elections/2023/governance-committee-election.md:20]: When I read that the deadline is 13 October, my mind registers that I can still submit my application on that date. So, the deadline would be 13 October 2023 23:59 UTC instead?

```suggestion
* If you've been nominated or are willing to nominate yourself: check the [charter document](../../governance-charter.md) and confirm you are ready for the commitment. Make sure to provide all necessary information before 13 October 2023 23:59 UTC
```

File: elections/2023/governance-committee-election.md [hunk 7]
```diff
@@ -0,0 +1,108 @@
...
+* If you are an active community member: confirm that you are on the voters list (see [election announcements issue](https://github.com/open-telemetry/community/issues/1561)) or register yourself before 20 October 2023 23:59 UTC.
+* Vote between 23 October 2023 00:00 UTC and 25 October 2023 23:59 UTC via the [voting link](https://vote.heliosvoting.org/helios/elections/1ee70ee4-11ce-11ee-aaf8-0a8c9aac83f9/view)
```
Comment (jpkrohling) [inline elections/2023/governance-committee-election.md:22]: Can you double-check the dates, verifying it matches Helios?

image

File: elections/2023/governance-committee-election.md [hunk 8]
```diff
@@ -0,0 +1,108 @@
...
+* [Daniel Dyla](https://github.com/dyladan), Dynatrace, until October 2024
+* [Juraci Paixão Kröhling](https://github.com/jpkrohling), Grafana Labs, until October 2023
+* [Morgan McLean](https://github.com/mtwo), Splunk, until October 2024
+* [Ted Young](https://github.com/tedsuo), Lightstep, until October 2023
+* [Trask Stalnaker](https://github.com/trask), Microsoft, until October 2024
+* [Yuri Shkuro](https://github.com/yurishkuro), Facebook, until October 2023
+
+Five people must be elected in this election, each with two-year terms. Note that five current Governance Committee members end terms as part of this election cycle, though they are welcome to run again for one of the five vacancies.
+
+After this election and until the next election, the Governance Committee will have nine seats, all of which will be elected positions. Please refer to the [charter document](../../governance-charter.md#establishment-of-a-governance-committee) for more information.
```
Comment (jpkrohling) [inline elections/2023/governance-committee-election.md:41]: It's time to remove this paragraph. It's from a time when we were changing the size of the GC.

File: elections/2023/governance-committee-election.md [hunk 9]
```diff
@@ -0,0 +1,108 @@
...
+
+Everyone with voting rights may log into [Helios Voting](https://vote.heliosvoting.org/helios/elections/1ee70ee4-11ce-11ee-aaf8-0a8c9aac83f9/view) using their GitHub account. Voting will be approval voting, where each voter may select up to five candidates. The five candidates with the most votes win the election.
+
+The election committee will accept late registrations to vote and requests to re-send the voting link via email to [cncf-opentelemetry-governance@lists.cncf.io](mailto:cncf-opentelemetry-governance@lists.cncf.io). We encourage pre-registration to minimize the effort required to run this election.
+
+Per Helios, voting is entirely private: nobody will know any individual's vote.
+
+# Results
+
+Voting will close on 25 October 2023 at 00:00 UTC. Nominees will be stack ranked. If a nominee becomes ineligible (for instance, if more than three topmost nominees work for the same company), the election committee will skip those nominees and pick the nominee with the next-highest score. The exact scores for each candidate will be public.
```
Comment (jpkrohling) [inline elections/2023/governance-committee-election.md:92]:
```suggestion
Voting will close on 25 October 2023 at 23:59 UTC. Nominees will be stack ranked. If a nominee becomes ineligible (for instance, if more than three topmost nominees work for the same company), the election committee will skip those nominees and pick the nominee with the next-highest score. The exact scores for each candidate will be public.
```

**Review by jpkrohling — APPROVED** [review]

**Comment by jpkrohling** [comment]
I would still like to have @trask's approval before merging this, given he's at the election committee.

**Review by trask — APPROVED** [review]

**Pull request merged**